### PR TITLE
improve(rds): fix rds resource alicloud_db_backup_policy`s log_backup bug

### PR DESCRIPTION
**Pull request by zhangyuangang**
image
image
image
image

### CODE REVIEW

File: website/docs/r/db_backup_policy.html.markdown [hunk 1]
```diff
@@ -62,8 +62,11 @@ The following arguments are supported:
...
 * `backup_time` - (Optional) DB instance backup time, in the format of HH:mmZ- HH:mmZ. Time setting interval is one hour. Default to "02:00Z-03:00Z". China time is 8 hours behind it.
 * `retention_period` - (Optional) Instance backup retention days. Valid values: [7-730]. Default to 7.
-* `log_backup` - (Optional) Whether to backup instance log. Note: The 'Basic Edition' category Rds instance does not support setting log backup. [What is Basic Edition](https://www.alibabacloud.com/help/doc-detail/48980.htm).
-* `log_retention_period` - (Optional) Instance log backup retention days. Valid when the `log_backup` is `true`. Valid values: [7-730]. Default to 7. It cannot be larger than `retention_period`.
+* `enable_backup_log` - (Optional) Whether to backup instance log. Valid values are `1`, `0`, Default to `1`. Note: The 'Basic Edition' category Rds instance does not support setting log backup. [What is Basic Edition](https://www.alibabacloud.com/help/doc-detail/48980.htm).
```
Comment (Lexsss) [inline website/docs/r/db_backup_policy.html.markdown]: available in 1.67.0+

File: alicloud/resource_alicloud_db_backup_policy.go [hunk 2]
```diff
@@ -68,6 +69,27 @@ func resourceAlicloudDBBackupPolicy() *schema.Resource {
...
+				Type:             schema.TypeInt,
+				ValidateFunc:     validation.IntBetween(5, 50),
+				Optional:         true,
+				DiffSuppressFunc: logRetentionPeriodDiffSuppressFunc,
+			},
+
+			"high_space_usage_protection": {
+				Type:             schema.TypeString,
+				ValidateFunc:     validation.StringInSlice([]string{"Enable", "Disable"}, false),
+				Optional:         true,
```
Comment (Lexsss) [inline alicloud/resource_alicloud_db_backup_policy.go]: default = ？

Comment (zhangyuangang) [reply alicloud/resource_alicloud_db_backup_policy.go]: > default = ？

done

File: website/docs/r/db_backup_policy.html.markdown (hunk 1 above)
Comment (zhangyuangang) [reply website/docs/r/db_backup_policy.html.markdown]: > available in 1.67.0+

done

**Pull request merged**